src/buffer/out/textBuffer.hpp

```diff
@@ -229,6 +233,9 @@ class TextBuffer final
 
     void _PruneHyperlinks();
 
+    std::unordered_map<size_t, std::wstring> _IdsAndPatterns;

```

Comment:
Why wasn't this one `_idsAndPatterns`? Don't we usually have underscore and lowercase starting private variable names?